tests/inputs/enum/test_enum.py

```diff
@@ -82,3 +79,8 @@ def enum_generator():
         yield Choice.THREE
 
     assert Test(choices=enum_generator()).to_dict()["choices"] == ["ONE", "THREE"]
+
+
+def test_renamed_enum_members():
+    for member in ArithmeticOperator:
```

Comment:
```suggestion
    assert set(ArithmeticOperator) == set(("NONE", "PLUS", "MINUS")):
```

Reply:
That will fail due to sets sorting keys but I get what you're going for